packages/core/web-reporter-ui/src/components/Charts/ReportChart.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useMemo, useContext } from "react";
+import { VideoEnabledContext } from "../../../videoCurrentTimeContext";
+import { ApexOptions } from "apexcharts";
+import { getColorPalette } from "../../theme/colors";
+import { getLastX, useSetVideoTimeOnMouseHover } from "./useSetVideoTimeOnMouseHover";
+import { AnnotationInterval, LineSeriesType } from "./types";
+import { getAnnotations } from "./getAnnotations";
+import { Chart } from "./Chart";
+
+export const ReportChart = ({
```

Comment:
I think you need to export it in `index.ts` to use it in cloud version right?

Reply:
No need for now because all the charts that use it are declared in the open source version

Reply:
oh right we use the ReporterView directly 👌